src/wrapt/_wrappers.c

```diff
@@ -1825,12 +1646,10 @@ static PyMethodDef WraptObjectProxy_methods[] = {
     { "__bytes__",  (PyCFunction)WraptObjectProxy_bytes, METH_NOARGS, 0 },
     { "__format__",  (PyCFunction)WraptObjectProxy_format, METH_VARARGS, 0 },
     { "__reversed__", (PyCFunction)WraptObjectProxy_reversed, METH_NOARGS, 0 },
-#if PY_MAJOR_VERSION >= 3
     { "__round__",  (PyCFunction)WraptObjectProxy_round,
                     METH_VARARGS | METH_KEYWORDS, 0 },
-#endif
     { "__complex__",  (PyCFunction)WraptObjectProxy_complex, METH_NOARGS, 0 },
-#if PY_MAJOR_VERSION > 3 || (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 7)
+#if PY_MINOR_VERSION >= 7
```

Comment:
Even though this section covers into Python 3 I would clean it up now else might be confusing when come back to to see just a minor version check.